fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/MetricUnionCompositeTableDefinition.java

```diff
@@ -93,7 +99,12 @@ public PhysicalTable build(ResourceDictionaries dictionaries, DataSourceMetadata
     private Set<PhysicalTable> getPhysicalTables(ResourceDictionaries resourceDictionaries) {
         PhysicalTableDictionary physicalTableDictionary = resourceDictionaries.getPhysicalDictionary();
         return dependentTableNames.stream()
-                .map(name -> physicalTableDictionary.getOrDefault(name, null))
+                .peek(name -> {
```

Comment:
I'd leave this check in here. It only happens at table load time, and it's better to warn about any that are found missing, rather than not I think.